src/mixins/numeral-formatable.js

```diff
@@ -0,0 +1,278 @@
+import {
+  allowedMasks
+} from '../util/mask'
+
+export default {
+  data: () => ({
+    options: {},
+    signSymbol: '',
+    preDefined: {
+      'numeral': {
+        positiveOnly: false, // Positive value only
+        positiveSign: false, // Show positive sign '+'
+        blockSeparator: ' ', // {String|Array} - Separate between groups of digits
+        blockSize: 3, // {Number|Array} - Number of digits per group
+        decimal: '.', // Decimal mark
+        prefix: '', // Prefix shown in input field
+        precision: 2, // Floating point precision
+        suffix: '', // Suffix shown in input field
+        formatter: 'numeral'
+      }
+    }
+  }),
+
+  computed: {
+    precision () {
+      return this.absVal(this.options.precision)
+    },
+    positiveSign () {
+      return this.options.positiveSign ? '+' : ''
+    },
+    decimal () {
+      return this.oneChar(this.options.decimal)
+    },
+    blockSeparator () { // Default separator = ' '
+      const separator = this.options.blockSeparator
+
+      if (!Array.isArray(separator)) return [this.oneChar(separator, ' ')]
+
+      switch (separator.length) {
```

Comment:
Looks like it's forcing block separator, what if I don't want to have a block separator?

Reply:
I forgot that no-separator condition 😬